### PR TITLE
Update CHIPS docs

### PR DESCRIPTION
**Pull request by mihajlija**
Changes proposed in this pull request:

- Update OT duration
- Remove `Accept-CH: Sec-CH-Partitioned-Cookies` OT requirement

cc @dcthetall @krgovind

### CODE REVIEW

File: site/en/blog/chips-origin-trial/index.md [hunk 1]
```diff
@@ -22,17 +32,15 @@ This trial is available as a [third-party origin trial](/blog/third-party-origin
...
+The trial will be available from Chrome 100 to Chrome 105. Check the [Chrome release schedule](https://chromiumdash.appspot.com/schedule) for the planned release dates.
 
 ## How to enroll in the origin trial
 
 ### Prerequisites
 
-Chrome stable 100.
+Chrome stable 103.
 
 ### Steps
```
Comment (DCtheTall) [inline site/en/blog/chips-origin-trial/index.md:45]: The `Origin-Trial` header should only be included in responses with `Set-Cookie` that include `Partitioned`, please update step 2.

File: site/en/blog/chips-origin-trial/index.md [hunk 2]
```diff
@@ -131,7 +120,7 @@ Not every client will have the origin trial enabled.
 
 ### Cookies requirements
 
--   Partitioned cookies must be set with the `Secure` and `Path=/` and without the `Domain` attribute. 
+-   Partitioned cookies must be set with the `Secure` and `Path=/` and without the `Domain` attribute.
```
Comment (DCtheTall) [inline site/en/blog/chips-origin-trial/index.md:123]: Nit: the no-Domain requirement is not going to be enforced in M104, though it is TBD if this change is permanent.

File: site/en/docs/privacy-sandbox/chips/index.md [hunk 3]
```diff
@@ -7,14 +7,17 @@ description: >
...
   - mihajlija
+tags:
+  - cookies
+  - privacy  
 ---
 
 ## Implementation status
 
-- [Origin trial](/origintrials/#/view_trial/1239615797433729025) available from Chrome 100 to 103
+- [Origin trial](/origintrials/#/view_trial/1239615797433729025) available from Chrome 100 to 105
```
Comment (DCtheTall) [inline site/en/docs/privacy-sandbox/chips/index.md:20]: Nit: "Chrome 103 to 105"

**Review by DCtheTall — COMMENTED** [review]
Overall looks good, some comments though (I do not have permission to actually request changes 😬)

File: site/en/blog/chips-origin-trial/index.md [hunk 4]
```diff
@@ -6,10 +6,20 @@ subhead: >
...
+  - mihajlija  
+tags:
+ - origin-trials
+ - cookies
+ - privacy  
 ---
 
+## Changes
+
+- **May 2022**: As of Chrome 103, sending `Accept-CH: Sec-CH-Partitioned-Cookies` header is no longer required for optting into the origin trial.
```
Comment (mihajlija) [inline site/en/blog/chips-origin-trial/index.md:20]:
```suggestion
- **May 2022**: As of Chrome 103, sending `Accept-CH: Sec-CH-Partitioned-Cookies` header is no longer required for opting into the origin trial.
```

File: site/en/blog/chips-origin-trial/index.md (hunk 2 above)
Comment (mihajlija) [reply site/en/blog/chips-origin-trial/index.md:121]: I'm gonna edit this once the change lands in 104

**Review by DCtheTall — COMMENTED** [review]
LGTM

**Pull request merged**